src/test/java/net/datafaker/base/OverwatchTest.java

```diff
@@ -1,10 +1,11 @@
 package net.datafaker.base;
```

Comment:
Also makes sense to move to `package net.datafaker.videogame;`

Reply:
Fixed
Forgot about that

Reply:
@snuyanzin  Feel free to merge if everything seems okay